tests/e2e/bootc-install.sh

```diff
@@ -76,35 +51,64 @@ EOF
             REDHAT_VERSION_ID="41"
         fi
         SSH_USER="fedora"
-        ADD_REPO=""
+        BOOT_ARGS="uefi"
         ;;
     *)
         redprint "Variable TEST_OS has to be defined"
         exit 1
         ;;
 esac
 
-sed "s/REPLACE_ME/${QUAY_SECRET}/g" files/auth.template | tee auth.json > /dev/null
+# Setup local registry
+greenprint "Generate certificate"
+openssl req \
+    -newkey rsa:4096 \
+    -nodes \
+    -sha256 \
+    -keyout "${TEMPDIR}/domain.key" \
+    -addext "subjectAltName = IP:${REGISTRY_IP}" \
+    -x509 \
+    -days 365 \
+    -out "${TEMPDIR}/domain.crt" \
+    -subj "/C=US/ST=Denial/L=Stockholm/O=bootc/OU=bootc-test/CN=bootc-test/emailAddress=bootc-test@bootc-test.org"
+
+greenprint "Update CA Trust"
+sudo cp "${TEMPDIR}/domain.crt" "/etc/pki/ca-trust/source/anchors/${REGISTRY_IP}.crt"
+sudo update-ca-trust
+
+greenprint "Deploy local registry"
+podman run \
```

Comment:
90% of the commands in this file start with `sudo`...but this one doesn't. Is that intentional?

I've been caught out in the past by being confused in similar CI jobs mixing things that require `sudo` vs those that don't and wondering why my `podman build` isn't visible.

It seems to me it'd be simpler to just run everything here as root - and where we specifically want to lower privileges, we can do so via e.g. `systemd-run -P DynamicUser=yes` for example.

Reply:
Updated.